File: Latihan b/Python/memory.py
# Include/import
from hardware import hardware
# deklarasi kelas memori inherit hardware
class memory(hardware):
    # method dan konstruktor
    def __init__(self):
        self.frequency = 0
        self.memorysize = 0
        self.supportscuda = 0
    #  get dans set semua variabel
    def setfrequency(self, frequency):
        self.frequency = frequency
    def getfrequency(self):
        return self.frequency
    def setmemorysize(self, memorysize):
        self.memorysize = memorysize
    def getmemorysize(self):
        return self.memorysize
    def setsupportscuda(self, supportscuda):
        self.supportscuda = supportscuda
    def getsupportscuda(self):
        return self.supportscuda

File: Latihan b/cpp/memory.cpp
// Include
#include <iostream>
#include <string>
#include "hardware.cpp"
using namespace std;

// deklarasi kelas memori inherit hardware
class memory : public hardware{
    // variabel privat
    private:
        int frequency;
        int memorysize;
        int supportcuda;
    // method public dan konstruktor
    public:
        memory(){

        }
        // get dans set semua variabel
        void setfrequency(int frequency){
            this->frequency = frequency;
        }
        int getfrequency(){
            return this->frequency;
        }
        void setmemorysize(int memorysize){
            this->memorysize = memorysize;
        }
        int getmemorysize(){
            return this->memorysize;
        }
        void setsupportcuda(int supportcuda){
            this->supportcuda = supportcuda;
        }
        int getsupportcuda(){
            return this->supportcuda;
        }
        // desktruktor
        ~memory(){
        }
};